tools/manifest-templator/manifest-templator.go

```diff
@@ -114,6 +114,10 @@ func main() {
 		GeneratedManifests: make(map[string]string),
 	}
 
+	if featureGates != nil && strings.Contains(*featureGates, "NonRoot") {
+		*featureGates = *featureGates + ",NonRootExperimental"
+	}
+
```

Comment:
what's going on here? i see this is temporary, but what happened?

Reply:
This is based on https://github.com/kubevirt/kubevirt/pull/6112 and therefore you see unrelated stuff.